tests/benchmarks/bench_assembly.cpp

```diff
@@ -0,0 +1,389 @@
+// Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "bench.hpp"
+
+#ifdef MFEM_USE_BENCHMARK
+
+/*
+  This benchmark contains the implementation of the CEED's bake-off problems:
```

Comment:
Is the goal of the PR to re-write the CEED's bake-off problems/kernels or add the assembly benchmarks?

Reply:
Not necessarily, but it could be, it is just a benchmark that I find useful because it is testing all the different levels of assembly whereas the ceed benchmark focuses on PA.